zkevm-circuits/src/evm_circuit.rs

```diff
@@ -356,6 +370,14 @@ impl<F: FieldExt> EvmCircuit<F> {
             meta.advice_column(), // val4
         ];
 
+        let bytecode_table = [
+            meta.advice_column(), // code_hash
+            meta.advice_column(), // index
+            meta.advice_column(), // push_rindex
```

Comment:
I don't think there's a need to make `push_rindex` part of the lookup data, it seems to me `is_code` should be sufficient for all use cases of the bytecode table? 

I think we do need to expose `code_size` because we'll need it for e.g. `EXTCODESIZE` and very likely even for the error cases of `JUMP`.

Reply:
thanks for reviewing, I think push_rindex is auxiliary field helping for constraint byte code table cells, which enhance byte code circuit validity checking,  about this refer to https://hackmd.io/jM3A6PdlRESOpq4KaAqSZQ  han and i initial ideas,    code_size seems be required later as current stage only handle jump successful case.

Reply:
My thought is also to _not_ include the `push_rindex`, since that's for Bytecode circuit itself as auxiliary witness. (I seemed to have a typo in the note, just fixed)

Reply:
removing make it simper & updated , thanks